### PR TITLE
feat(install): add Windows install script

### PR DESCRIPTION
**Pull request by bryanoltman**
## Description

Adds an installer script for Windows, mirroring the functionality of `install.sh`.

https://github.com/shorebirdtech/shorebird/pull/369 will need to be merged first

The final step after this will be to update the readme with a command to download and run this script.

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: install.ps1 [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+if (Test-Path $installDirectory) {
+    Write-Output "Shorebird is already installed."
+    return
+}
+
+Write-Output "Installing Shorebird to $installDirectory..."
+
+# & git clone https://github.com/shorebirdtech/shorebird.git -b stable $installDirectory
+& git clone https://github.com/shorebirdtech/shorebird.git -b bo/flutter-batch-files $installDirectory
```
Comment (felangel) [inline install.ps1]: This is just for testing I assume and we'll revert before merging?

File: install.ps1 [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Write-Output @"
+🐦 Shorebird has been installed!"
+
+To get started, add $installDirectory\bin to your PATH environment variable and run the following command:
+
+shorebird --help
+
+For more information, visit:
+https://github.com/shorebirdtech/shorebird/blob/main/TRUSTED_TESTERS.md.
```
Comment (felangel) [inline install.ps1]:
```suggestion
https://docs.shorebird.dev
```

😉

**Review by felangel — APPROVED** [review]
LGTM but I don't have a windows machine to test on atm

File: install.ps1 (hunk 1 above)
Comment (bryanoltman) [reply install.ps1]: Correct, once https://github.com/shorebirdtech/shorebird/pull/369 lands in stable, I'll uncomment the line above and delete this one.

File: install.ps1 (hunk 2 above)
Comment (bryanoltman) [reply install.ps1]: lol, fixed

File: install.ps1 (hunk 1 above)
Comment (bryanoltman) [reply install.ps1]: Done, although this won't work until we update `stable`

**Pull request merged**